.github/workflows/old/issue-analyzer.yml

```
@@ -0,0 +1,44 @@
+      issue_body:
+        description: 'Issue Body'
+        required: true
+
+jobs:
+  analyze-issue:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)